server/activate_hooks.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
I'm not sure if this file is really needed. `checkServerVersion` is a relikt from the past.

Reply:
Okay, I'm not totally aware. Looking at other, seemed like I should be using `mattermost-plugin-api`. But if it's good, I'll leave this as is and will update per new approach on separate PR.  Any thoughts?

Reply:
Agree with @hanzei -- I think the only thing needed in this file is:
```
func (p *Plugin) OnActivate() error {
	if err := p.IsValid(); err != nil {
		return err
	}

	p.API.RegisterCommand(getCommand())

	return nil
}
```